### PR TITLE
Performance Testing Setup Script

### LINKED ISSUE #6813: Create performance testing kit
- [x] 1000 orders per second (in one market and 100/s across 10 markets)
 - [x] 100s of markets on the site
 - [x] test with user accounts that've traded in 100s of markets
 - [x] and with accounts that've done 1000s of trades and have 100+ open positions

### PR DESCRIPTION
**Pull request by robert-e-davidson3**
Fixes #6813

### CODE REVIEW

File: packages/augur-core/source/libraries/HelperFunctions.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ export async function sleep(milliseconds: number): Promise<void> {
     return new Promise<void>(resolve => setTimeout(resolve, milliseconds));
 }
 
-export async function resolveAll(promises: Iterable<Promise<any>>) {
+export async function resolveAll<T>(promises: Iterable<Promise<T>>) {
     let firstError: Error|null = null;
-    for (let promise of promises) {
+    for (const promise of promises) {
```
Comment (pgebheim) [inline packages/augur-core/source/libraries/HelperFunctions.ts:11]: Why is this serial?

File: packages/augur-tools/src/flash/scripts.ts [hunk 2]
```diff
@@ -2002,6 +1995,18 @@ export function addScripts(flash: FlashSession) {
         description: 'Do not stop dockers after running command and do not wait for user input before exiting',
         flag: true,
       },
+      {
+        name: 'do-not-run-geth',
+        abbr: 'G',
```
Comment (pgebheim) [inline packages/augur-tools/src/flash/scripts.ts:2008]: Love this

**Comment by robert-e-davidson3** [comment]
> Ooo now now all of these are settable during webpack build?

Yup. The big one was GSN_ENABLED since our perf testing doesn't use GSN, to avoid it as as bottleneck. But now you can set whichever without needing to modify the environment file.

**Review by pgebheim — APPROVED** [review]

File: packages/augur-core/source/libraries/HelperFunctions.ts (hunk 1 above)
Comment (robert-e-davidson3) [reply packages/augur-core/source/libraries/HelperFunctions.ts:11]: It isn't serial, actually. The promises are already executing. Awaiting them just makes sure each one finishes.

**Pull request merged**